gw_spaceheat/actors/actor_base.py

```diff
@@ -36,13 +37,11 @@ def __init__(self, node: ShNode):
             self.consume_client.on_log = self.on_log
         self.consume_client.subscribe(list(map(lambda x: (f"{x.Topic}", x.Qos.value), self.subscriptions())))
         self.consume_client.on_message = self.on_mqtt_message
```

Comment:
Setting callbacks should happen before anything "live" is started, i.e., before connect().

Basic operation should probably be something like: 
In ActorBase __init__(): 
  Construct client
  Change config (e.g. set on_message or password etc)

In ActorBase.start():
  client.connect()
  client.subscribe()
  client.start_loop()